packages/frontend/.gitignore

```diff
@@ -14,4 +14,7 @@ build/
 
 # Ignore environment variables
 .env
-config/*.json
\ No newline at end of file
+config/*.json
+
+src/admin-pages/settings.json
+src/public-pages/settings.json
```

Comment:
We also need to add `config/custom.css` as ignored here.  It shows up as an untracked file after e.g. `npm run frontend:build`.